osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -191,9 +192,12 @@ public ChatOverlay()
         }
 
         private double startDragChatHeight;
+        private bool canBeResized;
 
         protected override bool OnDragStart(InputState state)
         {
+            canBeResized = tabsArea.IsHovered;
+
             if (!channelTabs.IsHovered)

```

Comment:
did you miss this line? should probably be worked into your variable. also shouldn't be called `canBeResized`, it should be `isDragging` or something.